### PR TITLE
AArch64: Implement Java code for DDR

### PR DESCRIPTION
**Pull request by knn-k**
This commit implements Java code for DDR for AArch64.

Signed-off-by: KONNO Kazuhiro <konno@jp.ibm.com>

### CODE REVIEW

**Comment by knn-k** [comment]
Not tested, but this PR does mostly the same as what #7441 does for RISC-V.

**Comment by pshipton** [comment]
@keithc-ca

**Comment by 0xdaryl** [comment]
@keithc-ca , would you mind reviewing this PR please?

File: jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/corereaders/NewElfDump.java [hunk 1]
```diff
@@ -469,8 +470,44 @@ public Address getLinkRegisterFrom(Map registers) {
...
+		public Address getInstructionPointerFrom(Map registers) {
+			return (Address) registers.get("pc");
+		}
+
+		public Address getLinkRegisterFrom(Map registers) {
+			return (Address) registers.get("lr");
+		}
+
+		public long readUID(ElfFile file) throws IOException {
+			return file.readInt() & 0xffffffffL;
```
Comment (keithc-ca) [inline jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/corereaders/NewElfDump.java]: This seems inconsistent when compared to `ELFAArch64DumpReader.readUID()` - please explain why they should not read a value of the same width.

Comment (knn-k) [reply jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/corereaders/NewElfDump.java]: Thank you for pointing that out.
It seems I wrote the `ELFAArch64DumpReader` code based on `ELFARM32DumpReader`, and I didn't notice the inconsistency at that time.

**Comment by keithc-ca** [comment]
Jenkins test sanity plinux jdk11

**Pull request merged**